src/components/LoginComponent/Login.js

```diff
@@ -0,0 +1,74 @@
+import {React, useState} from 'react';
+import Bar from '../BarComponent/Bar';
+
+const buttonStyle = {
+    borderRadius: '40px',
+    height: '20%',
+    width: '50%',
+    margin: '1vh',
+    fontFamily: 'Anonymous Pro',
+    fontSize: '3vw',
+    background: '#ADADAD',
+    border: 'none',
+  }
+
+function Login(){
+
+    const [username, getUser] = useState('');
```

Comment:
change the two states and make it one state there's no point in having 2 states because it'll be harder later to check it when making the login system